api/task.go

```diff
@@ -71,6 +71,10 @@ type TaskDatabasePITRRestorePayload struct {
 	// After the PITR operations, the database will be recovered to the state at this time.
 	// Represented in UNIX timestamp in seconds.
 	PointInTimeTs int64 `json:"pointInTimeTs,omitempty"`
+	// The instance owning the target database.
+	DstInstance *Instance `json:"dstInstance,omitempty"`
+	// The name of the target database, should only be used when DstInstance is not nil.
+	DstDatabase string `json:"DstDatabasem,omitempty"`
```

Comment:
This seems unnecessary because we have a CreateDatabaseContext containing enough information for creating a database in the target instance.